### ISSUE TITLE
Allow for passing in styles as a string instead of a url

### ISSUE THREAD

**Issue by SamGraber** [jamesshore/quixote#39]
I think it would be helpful if you could pass in styles as a string instead of giving quixote a stylesheet url. This would give users greater flexibility for how they pull in stylesheets, as they could use `require()` or something similar to import the styles and then pass them into quixote directly.

We are considering quixote for style testing in our project. Currently, our project uses less, but we'll shortly be migrating to sass instead. I'm thinking we could use a bundler like webpack to load our less/scss files and process them into css before passing the styles into quixote. This would be easier to do if we could pass them in directly instead of letting quixote load them.

**Comment by jamesshore** [comment]
Makes sense! If you'd like this to happen sooner, send me a pull request. The file that controls this is `src/q_frame.js` (and `src/_q_frame_test.js`). See CONTRIBUTING.md in the top level for instructions on building the code.

I'm imagining a new `css` parameter in the QFrame options object, like this:

```javascript
	it("creates iframe using stylesheet and source URL simultaneously", function(done) {
			var options = {
				src: "/base/src/_q_frame_test.html",
				css: ".style-me { font-size: 42px; }"
			};

			QFrame.create(window.document.body, options, function(err, frame) {
				var styleMe = frame.get(".style-me");
				assert.equal(styleMe.getRawStyle("font-size"), "42px");
				done();
			});
		});
	});
```

As of [the current version](https://github.com/jamesshore/quixote/blob/0b3b0d137d04695d88fab8e2c798e25d058b081e/src/_q_frame_test.js), the place to add the tests is line 121 in `_q_frame_test.js`. The code to modify in `q_frame.js` is `onFrameLoad()`, probably after the call to `loadStylesheets()`. (After line 64 of [the current version](https://github.com/jamesshore/quixote/blob/0b3b0d137d04695d88fab8e2c798e25d058b081e/src/q_frame.js).)

**Comment by SamGraber** [comment]
https://github.com/jamesshore/quixote/pull/40

**Comment by jamesshore** [comment]
Blocked on 0.13 release.

**Comment by jamesshore** [comment]
Integrated and will be included in v0.13 release.

**Comment by jamesshore** [comment]
Released in v0.13.
